src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector.cs

```diff
@@ -364,6 +413,12 @@ public override BoundStatement InstrumentReturnStatement(BoundReturnStatement or
                 return new BoundSequencePointWithSpan(original.Syntax, rewritten, ((BlockSyntax)original.Syntax).CloseBraceToken.Span);
             }
 
+            if (original.Syntax is ParameterSyntax parameterSyntax)
```

Comment:
I didn't follow this case. It seems it would apply to any return statement involving a parameter.

Reply:
The only return statement that is itself associated with a parameter (as opposed to having an expression that is associated with a parameter) is the one created for synthesized property getter. I can add an assert.